src/test/java/com/flightstats/hub/dao/aws/DynamoChannelConfigDaoTest.java

```diff
@@ -1,27 +1,61 @@
 package com.flightstats.hub.dao.aws;
 
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.model.AttributeValue;
+import com.amazonaws.services.dynamodbv2.model.GetItemRequest;
+import com.amazonaws.services.dynamodbv2.model.GetItemResult;
+import com.amazonaws.services.dynamodbv2.model.ScanRequest;
+import com.amazonaws.services.dynamodbv2.model.ScanResult;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import com.flightstats.hub.app.HubProvider;
+import com.flightstats.hub.config.DynamoProperties;
 import com.flightstats.hub.model.ChannelConfig;
 import com.flightstats.hub.test.Integration;
-import com.google.inject.Injector;
+import lombok.SneakyThrows;
 import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static junit.framework.Assert.assertNull;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasItems;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
+@ExtendWith(MockitoExtension.class)
 class DynamoChannelConfigDaoTest {
 
     private static final Logger logger = LoggerFactory.getLogger(DynamoChannelConfigDaoTest.class);
     private static DynamoChannelConfigDao channelConfigDao;
+    @Mock private AmazonDynamoDB dbClient;
+    @Mock private DynamoProperties dynamoProperties;
```

Comment:
Do we like that better?  I kinda prefer it on a single line so I can clearly see which of my ivars aren't real.

Reply:
(Historically) I think we mostly like IntelliJ auto-format and not having to configuring it, more than the actual format.

Reply:
I'd go with whatever IntelliJ auto-formats to. In the Dev Services code base, I habitually auto-reformat the file every time I finish editing some code so any code out of conformance ends up "fixed" anyway.

For the record, I'm with dkleckler, I like the one-liner for this, but it's worth it to stick to the norms if it means never worrying about auto-formatting screwing up some hand-format.

Of course, you can decide as a group what IntelliJ settings to override, just make sure everyone has the same settings (and it might not be possible to configure IntelliJ for granular exceptions, like in this case).

Reply:
Switched.